.github/workflows/build-and-deploy.yml

```diff
@@ -442,3 +442,30 @@ jobs:
           environment: ${{ matrix.environment }}
           sha: ${{ needs.test.outputs.IMAGE_TAG }}
           slack-webhook: ${{ secrets.SLACK_WEBHOOK }}
+
+  deploy-secondary:
+    name: Parallel Deployment
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    environment:
+      name: ${{ matrix.environment }}
+      url: ${{ steps.deploy_app_parallel.outputs.deploy-url }}
+    needs: [lint, test, deploy-main]
```

Comment:
deploy-main already depends on `[lint, test]` so we should be able to only say `[deploy-main]` here

Reply:
done